code-of-conduct/guide-code-of-conduct-response-plan.md

```diff
@@ -0,0 +1,188 @@
+  - `guide-code-of-conduct-incident-record.md`
+  - `guide-code-of-conduct-remediation-record`
+
+You might also consider using the `../our-team/template-our-team.md` template to let your community members know who they can contact to report a Code of Conduct violation. This document is not only necessary for Code of Conduct violations. It is a core document that helps you clearly communicate who belongs to your open source project or organization.
+
+
+## Why do I need a Code of Conduct and a response plan?
+
+See the `guide-code-of-conduct.md` for reasons to incorporate a Code of Conduct and response plan.
+
+
+## Content of the Code of Conduct response plan template
+
+The following sections provide guidance about how to fill out each section of the Code of Conduct response plan template.
+
+:information_source: The Code of Conduct response plan template includes boilerplate text that you can customize or adapt, use as-is, or completely replace with your own text. If you use the boilerplate text, make sure you replace the project name placeholders with your own project name.
+
+
+### About the "Introduction" section
+
+The introduction section explains the purpose of the response plan document.
+
+
+### About the "Community moderators" section
+
+In the *Community moderators* section, you can list the names of the community moderators or link to a document that lists the community moderators, such as a website that uses the (Our Team)[../our-team/template-our-team.md] template.
+
+Alternatively, you could list the names of the community moderators and their preferred contact information, such as their Slack handles.
+You could also link to a dedicated email address or contact form.
+
+Remember to respect your community moderator's privacy by only including contact methods that go through official project channels, such as community forums or project-issued email accounts.
+Don't include personal email addresses or contact information.
+The goal is to strike a balance between making your moderators easy to contact while also respecting their privacy.
+
+
+### About the "Contacting a community moderator" section
+
+In the *Contacting a community moderator* section, provide clear instructions for the best way to contact the community moderators.
+Filling out this section will require some pre-work on your part to clearly think through how community moderators should be contacted.
+
+
+### About the "Community moderator values" section
+
+In this section, articulate the values that your community moderators should strive to uphold.
+You can customize the provided boilerplate text or write your own.
+
+
+### About the "Requirements for community moderators" section
+
+In this section, explain the eligibility requirements for your community moderators.
+Once again, this section will require some pre-work for you to think through what your community moderators will need to do to be prepared to serve in this role:

```

Comment:
```suggestion
This section will require some pre-work for you to think through what your community moderators will need to do to be prepared to serve in this role:

```
suggestion: remove reference to prior mention. I don't think it adds much, and it doesn't aid scan-ability.